test/models/user_test.rb

```diff
@@ -39,9 +41,8 @@
     user.networks.must_include network
   end
 
-  it 'has a list of stories for approved accounts' do
-    start_count = user.individual_account.stories.count
-    create(:account, user: user, stories_count: 2)
-    user.approved_account_stories.count.must_equal start_count + 2
+  it 'has a list of stories and series' do
+    user.approved_account_stories.must_include story
```

Comment:
Are you leaving the User methods like `approved_account_stories`, even though they aren't being used anymore?